src/vault.py

```diff
@@ -104,6 +122,21 @@ def configure_kv_policy(self, policy: str, mount: str):
             mount_policy = fd.read()
         self._client.sys.create_or_update_policy(policy, mount_policy.format(mount=mount))
 
+    def enable_audit_device(self, device_type: str, path: str) -> None:
+        """Enable a new audit device at the supplied path."""
+        if device_type + "/" not in self._client.sys.list_enabled_audit_devices()["data"].keys():
+            if (
+                self._client.sys.enable_audit_device(
+                    device_type=device_type,
+                    options={"file_path": path},
+                ).status_code
+                != 204
+            ):
+                logger.error(
+                    "Failed to enable audit device of type: %s, using path: %s", device_type, path
+                )
+            logger.info("Enabled audit device of type: %s, using path: %s", device_type, path)
```

Comment:
Added it for now, but I'm hesitant about this, we will have this logged every time `on_config_changed` runs, and the user is not explicitly trying to enable the audit device. Wouldn't it make the logs a bit noisy?

Reply:
I think it's fine to log that audit device is enabled as info level.

Reply:
> I'm hesitant about this

Completely reasonably. If you think that's the case then lets ignore this for now. It was just something that came to mind, but I don't think I've been working with charms enough to understand the implications.

Reply:
This might even be a debug log. If you're concerned about noisyness, that's probably the best approach. Either way I'll leave the final call to you, I don't have a strong opinion here at all.